can/io/blf.py

```diff
@@ -584,9 +584,12 @@ def stop(self):
         """Stops logging and closes the file."""
         self._flush()
         if self.file.seekable():
-            filesize = self.file.tell()
-            # Write header in the beginning of the file
-            self.file.seek(0)
+            filesize = (
+                self.file.tell()
+            )  # == self.file.seek(0, 1) if type == gzip.GzipFile
+            if type(self.file) != gzip.GzipFile:
+                # Write header in the beginning of the file
+                self.file.seek(0)
```

Comment:
If the type of `self.file` is a `gzip.GzipFile`, then the `self.file.tell()` call appears to enter `gzip.seek(0, 1)`. Does it need to be called twice?